public/data/panam-py.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Hebron Panam Praise",
+  "type": "personal",
+  "bio": "Software Engineeer (Backend) with Python, Node.js and Go",
+  "avatar": "https://github.com/panam-py.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/panam-py",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/thisispanam",
+      "icon": "twitter"
+    }
+  ]
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

This will get fixed automatically by the linter